### PR TITLE
Create a hybrid case with both ble and usb component holes

### PR DESCRIPTION
**Pull request by endikallanomatxin**
I think it can be useful to have components mounted into the keyboards for both use cases. This case allows for that configuration.

It was made by intersecting ble and usb case parts (taking only the common).

Can anyone confirm if this would work? Is it possible to assemble components for both use cases in the same keyboard?

### CODE REVIEW

**Comment by endikallanomatxin** [comment]
Does anyone know if it is possible to assemble the components for both of BLE and USB? Can someone confirm that it works?

**Comment by shetty-tejas** [comment]
Hey @endikallanomatxin, I don’t really think that would work because there are two different variants of microcontrollers that can be used with the keyboard. You’d use (and solder) the RP2040 if you want the keyboard to be wired, and the other one if you want it to be BLE. Once it’s soldered, you won’t (and can’t) change how the keyboard functions, unless you open the keyboard and re-solder the required parts.
Given that limitation, why would you need a hybrid case if you can’t switch between wired and BLE modes as needed?

A hybrid case would only be useful if:

1. The TOTEM PCB and case support swapping microcontrollers (from RP2040 to the other) and have pluggable battery sockets, or
2. A single microcontroller could function as both BLE and wired, possibly via a switch or similar mechanism.

However, this is easier said than done, as BLE mode requires a battery, and wired mode requires a TRRS jack—using both simultaneously isn’t ideal anyway. Basically, an entire PCB revamp would be required to get this thing working.

So TL;DR: While this is a nice concept (and something I'd personally love to see), given the current state of the keyboard, it’s nearly useless.

**Comment by endikallanomatxin** [comment]
Hey @shetty-tejas, thanks for the clear, thorough and didactic answer! I didn't know the limitations you mentioned. Really appreciate your insights!

**Comment by rickard-von-essen** [comment]
Just noded that the [build guide](https://github.com/GEIGEIGEIST/TOTEM/blob/main/docs/images/buildguide/pcb_switches.jpg) depicts a build with both TRRS jack, power switch, and battery mounted.
